.changeset/event-bus-plugins.md

```
@@ -0,0 +1,150 @@
+const adapter = withPlugin(
+    new MemoryEventBusAdapter(),
+    withEventBusSchema({ eventMapSchema }),
+);
+const eventBus = new EventBus({ adapter });
+```
+
+### New Export Path
+
+A new export path has been added to `package.json`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)